ObsLocTAP.tex

```diff
@@ -563,13 +563,13 @@ \subsection{Observation Locator data model }
 removed from the planning table or they could return to the "Planned" status
 for a possible future schedule.
 
-Tracking type (\textbf{tracking\_type}) is an optional field of the obsplan table
+Tracking type (\textbf{tracking\_type}) is a field of the obsplan table
 to characterize the tracking foreseen during the observation execution.  
-If not specified, the default value would be \textbf{sidereal}. 
+If set to null, the default value would be \textbf{Sidereal}. 
```

Comment:
Is the intention to allow NULL as an alternative way to mean `Sidereal`?  That is a slight convenience for data providers, but a slight inconvenience for clients (who will have to test for `tracking_type IS [NOT] NULL` as well as the actual column value). It's also inconsistent with the other enumeration type columns which have the "not null" constraint.  Unless there are good reasons to do otherwise I would suggest to remove this sentence (or replace with something like "In most cases the value will be Sidereal"), and to add the "not null" constraint in Table 1.

Related: do you want to consider adding something like an "Other" option for this column? I have no idea whether the existing options cover all likely tracking type requirements.  Just a thought.